crates/evm/src/tests/queries/estimate_gas_tests.rs

```diff
@@ -363,6 +363,63 @@ fn estimate_gas_with_varied_inputs_test() {
     );
 }
 
+#[test]
+fn pending_env_test() {
```

Comment:
Convention `test_pending_env`

Reply:
I'll change that once CI passes and merge